src/components/Conversation/EmptyViewPointCard.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
```

Comment:
The component name technically should be EmptyViewPointSection and not card as this appears when the section is empty.